tests/unit_test/chat/user_media_test.py

```
@@ -1,316 +1,430 @@
-            "filename": self.filename,
-            "extension": self.extension,
-            "bot": self.bot,
-            "sender_id": self.sender_id,
-        }
 
-        class DummyMongo:
-            def to_dict(inner_self):
-                return data
+@pytest.mark.asyncio
+async def test_db_user_media_data():
+    os.environ["system_file"] = "./tests/testing_data/system.yaml"
```

Ruff (0.8.2)

Use capitalized environment variable `SYSTEM_FILE` instead of `system_file`

Replace `system_file` with `SYSTEM_FILE`

(SIM112)